VAMobile/src/store/api/api.ts

```diff
@@ -210,6 +210,24 @@ const call = async function <T>(
     // No errors found, return the response
     return await response.json()
   } else {
+    const overrideErrors = _store?.getState().demo.overrideErrors
+    if (overrideErrors) {
+      _.forEach(overrideErrors, (error) => {
+        if (error.endpoint === endpoint) {
+          throw error.error
+        }
+        if (endpoint.includes(error.endpoint)) {
+          throw error.error
+        }
+        if (
+          error.endpoint === '/v0/messaging/health/folders/${folderID}/messages' &&
```

Comment:
No this would not. The first part of this if is checking if the override error endpoint is that value. The other 2 parts are checking if the api called by our usage is /v0/messaging/health/folders/1/messages or a different folder. Both parts of this one is needed to not unintentionally throw another error for a different api.